wdl-analysis/tests/analysis.rs

```diff
@@ -110,7 +110,7 @@ fn compare_results(test: &Path, results: Vec<AnalysisResult>) -> Result<()> {
         // Attempt to strip the CWD from the result path
         let path: Cow<'_, str> = match &path {
             // Strip the CWD from the path
-            Some(path) => path.strip_prefix(&cwd).unwrap_or(&path).to_string_lossy(),
+            Some(path) => path.strip_prefix(&cwd).unwrap_or(path).to_string_lossy(),
```

Comment:
FWIW, this was a stray lint warning that clippy found and I just corrected here.